rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -251,6 +270,7 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
             CONVEX: (self._decode_fpis_claim, 'convex'),
             FOX_DISTRIBUTOR: (self._decode_fox_claim,),
             ELFI_LOCKING: (self._decode_elfi_claim,),
+            COW_PROTOCOL: (decode_cow_claim, self.vcow),
```

Comment:
This is the same as doing this
```suggestion
            self.vcow.evm_address: (decode_cow_claim, self.vcow),
```